react/src/components/lablupTalkativotUI/LLMPlaygroundPage.tsx

```diff
@@ -58,14 +73,23 @@ const LLMPlaygroundPage: React.FC<LLMPlaygroundPageProps> = ({ ...props }) => {
           align="stretch"
         >
           {_.map(list, (item, index) => (
-            <EndpointLLMChatCard
-              key={getKey(index)}
-              style={{ flex: 1 }}
-              onRequestClose={() => {
-                remove(index);
-              }}
-              closable={list.length > 1}
-            />
+            <Suspense
+              fallback={
+                <Card>
+                  <Skeleton active />
```

Comment:
When refreshed, it appears like this on the screen.
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/4393b6b9-dba4-4fdc-a6e1-a7e85fe2b5bb.png)

Reply:
Added `flex:1` to `Card` of `fallback`.